public/js/addhome.js

```diff
@@ -1,3 +1,29 @@
+const postFormHandler = async (event) => {
```

Comment:
we don't want to use the add home form here -- that already has a form in main, and you are trying to make the update route... needs a new js, or use the update.js